lib/ace/ext/papeeria/highlighter.coffee

```diff
@@ -5,11 +5,17 @@ define((require, exports, module) ->
     highlightBrackets = (editor) ->
         pos = findSurroundingBrackets(editor)
         session = editor.getSession()
+        cursorPosition = editor.getCursorPosition()
         if session.$bracketHighlightRight || session.$bracketHighlightLeft
             session.removeMarker(session.$bracketHighlightLeft)
             session.removeMarker(session.$bracketHighlightRight)
             session.$bracketHighlightLeft = null
             session.$bracketHighlightRight = null
+        if pos.mismatch && session.getLine(cursorPosition.row).charAt(cursorPosition.column - 1) of session.$brackets
```

Comment:
This is the case when a conflict occurs with standard highlight. `findOpenBracket` "swallow" "}" when {......}_ (_caret)